### PR TITLE
 add convert function to quill's clipboard api

### PR DESCRIPTION
**Pull request by mhamri**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/KillerCodeMonkey/ngx-quill/issues/77#issuecomment-343556041
- [?] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by AnielloFalcone — APPROVED** [review]
I can't find anything about it in the Docs even if I can find it in the quill.js file:
screenshot 2018-08-03 at 11 06 54

I suppose this looks good enough.

File: types/quill/index.d.ts [hunk 1]
```diff
@@ -42,6 +43,7 @@ export interface KeyboardStatic {
 }
 
 export interface ClipboardStatic {
+    convert(html: string|Blot): DeltaStatic;
```
Comment (43081j) [inline types/quill/index.d.ts]: am i looking at the wrong place [here](https://github.com/quilljs/quill/blob/4a6bd6a0fb77078bb280764259a78f7f28254e4a/modules/clipboard.js#L79)?

the method signature shows it takes an object with properties `html` and/or `text`. Not a string or `Blot` (unless a blot has those properties...).

presumably it somehow worked when you tried it so maybe im in the wrong part of the source?

**Comment by sheetalkamat** [comment]
@mhamri  This PR is waiting for merging for answer to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27830#discussion_r207571169

File: types/quill/index.d.ts (hunk 1 above)
Comment (mhamri) [reply types/quill/index.d.ts]: yes, the last release is 1.3.6, I think you are looking at the master branch

https://github.com/quilljs/quill/blob/1.3.6/modules/clipboard.js

Comment (mhamri) [reply types/quill/index.d.ts]: 1.3.6  is the latest release 
https://github.com/quilljs/quill/releases

**Comment by 43081j** [comment]
@sheetalkamat the signature isn't right in this... it should be `(html: string)` rather than `(html: string|Blot)` as per the source [here](https://github.com/quilljs/quill/blob/96e38e92637b75b907579d0cc1b201920aebe38c/modules/clipboard.js#L75).

The function there, as you can see, doesn't make use of the parameter unless it is a string. it looks like the only other type of call is a parameterless one (you can see they call `convert()` below). I don't see anywhere in the source or the docs which suggests you can pass a `Blot`.

Not like it'll break anything but it will be misleading if it is wrong.

**Comment by sheetalkamat** [comment]
Please create a separate PR to fix the signature in that case.

**Pull request merged**